01-node-tutorial/answers/prompter.js

```diff
@@ -28,6 +28,7 @@ let item = "Enter something below.";
 const form = () => {
   return `
   <body>
+  <label>Guess a number between 0 - 100</label>
   <p>${item}</p>
   <form method="POST">
   <input name="item"></input>
```

Comment:
Will the user be able to enter characters as well as number?

Reply:
I didn't add verification as I should have, so yes they could.